src/main/python/janusgraph_python/driver/ClientBuilder.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ..structure.io.GraphsonReader import JanusGraphSONReader
+from ..structure.io.GraphsonWriter import JanusGraphSONWriter
+
+from gremlin_python.driver.driver_remote_connection import DriverRemoteConnection
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONReader
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONWriter
+
+
+class JanusGraphClient(object):
+    """
+        JanusGraph Client Builder which adds the Serializers for JanusGraph specific objects, predicates etc.
+    """
+
+    REMOTE_CONNECTION = None
+
+    def __init__(self, version=3.0):
+        """
+            Initializing with GraphSON version 3.0
+        Args:
+            version (int):
+        """
+
+        self.graphsonVersion = version
+        pass
+
+    def connect(self, host="loclahost", port="8182", graph="g", **kwargs):
```

Comment:
This should be:
```
def connect(self, url="localhost", port="8182", graph="g", **kwargs):
```

Reply:
Agree on the misspelled `localhost`. I requested the change from `url` to `host` previously, as noted in the parameter doc below.

Reply:
My bad. Latest commit fixes this.